### PR TITLE
Add a bio for GitHubTeacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio for @githubteacher as requested in #3.

@githubstudent I would :heart: some :eyes: and :thought_balloon: on this when you have a chance.  :smiley:

### CODE REVIEW

**Comment by githubteacher** [comment]
This looks pretty good, but I need a few :clock1: to review everything.

File: docs/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## GitHub Teacher
+
+**Name:** GitHub Teacher
+
+**Role:** Teacher
```
Comment (githubteacher) [inline docs/githubteacher.md]: Maybe trainer is a better word here.

**Comment by abjaiswal** [comment]
Test  Code

**Comment by githubteacher** [comment]
:+1: 

:ship: it!

:shipit:

**Pull request merged**